### ISSUE TITLE
Verwijderde schakelaars /sensors

### ISSUE THREAD

**Issue by DavidVM1982** [Egregius/HomewizardPHP#34]
Als er  schakelaar verwijderd worden in HW bijven deze in de database staan.
ASUN elementen worden niet weer gegeven in home scherm wel in de settings.

ps: ben bijna klaar met opkuisen van de code. en zal dan mijn projectje delen. 
(was op ski vakantie)

**Comment by Egregius** [comment]
Correct, die moet je zelf wissen via bewerk schakelaar en bewerk sensoren. 
Ik kijk even voor de asun.

**Comment by Egregius** [comment]
Hmmm, kan je aanduiden in welke versie het wel nog werkte? 
De knoppen staan er al terug in v20150210 maar de commando's ben ik kwijt geraakt.

**Comment by DavidVM1982** [comment]
in 1 van de eerste versie was werkte deze functie

**Comment by Egregius** [comment]
Ik denk dat ASUN het terug moet doen in v20150213.
